Babylon/utils/decorators.py

```diff
@@ -39,6 +39,23 @@ def wrapper(*args: Any, **kwargs: Any) -> Any:
     return wrapper
 
 
+def output_to_file(func: Callable[..., Any]) -> Callable[..., Any]:
+    """Add output to file option to a command"""
+
+    @wraps(func)
+    @click.option("-o", "--output", "output_file", help="File to which content should be outputted (json-formatted)")
+    def wrapper(*args: Any, **kwargs: Any) -> Any:
+        output_file = kwargs.pop("output_file")
+        response = func(*args, **kwargs)
+        if not output_file:
```

Comment:
```suggestion
        if output_file:
            with open(output_file, "w") as _f:
                _f.write(response.toJSON())
        return response
```

Reply:
It's actually a coding style choice. I don't remember where I got that habit.
I reduce indent as much as possible for readability and I return as soon as I don't have anything to do
Reducing indent is a universal good practice for the rest I don't know, just a choice. I don't think it matters much, do you ?

Reply:
if you just want to reduce the indent we could go with :
```python
if output_file:
    _f = open(output_file,"w")
    _f.write(response.toJSON())
    _f.close()
return response
```

Reply:
it does not use context, which is in pep8 and good practice